README.md

```diff
@@ -78,8 +78,8 @@ humanizeDuration(260040000, { spacer: "" }); // '3days, 14minutes'
 Number representing the maximum number of units to display for the duration.
 
 ```js
-humanizeDuration(1000000000000); // '31 years, 8 months, 1 week, 19 hours, 46 minutes, 40 seconds'
-humanizeDuration(1000000000000, { largest: 2 }); // '31 years, 8 month'
+humanizeDuration(1000000000000); // '1653 weeks, 3 days, 1 hour, 46 minutes, 40 seconds'
```

Comment:
Could you change this to a smaller value? Maybe `10000000000` (which should humanize to "16 weeks, 3 days, 17 hours, 46 minutes, 40 seconds")?